WordPress/src/main/java/org/wordpress/android/ui/accounts/HelpActivity.kt

```diff
@@ -67,10 +69,13 @@ class HelpActivity : AppCompatActivity() {
                 emailSuggestion = supportHelper
                         .getSupportEmailAndNameSuggestion(accountStore.account, selectedSiteFromExtras).first
             }
+
             supportHelper.showSupportIdentityInputDialog(this, emailSuggestion, isNameInputHidden = true) { email, _ ->
                 zendeskHelper.setSupportEmail(email)
                 refreshContactEmailText()
+                AnalyticsTracker.track(Stat.SUPPORT_IDENTITY_SET)
```

Comment:
@aerych This PR is fine as is, but I have a couple minor suggestions.

I think it would be better if we had this in `ZendeskHelper.setSupportEmail` since it's where the identity change happens. I'd also suggest checking if the email is actually changed (comparing to `AppPrefs.getSupportEmail`). The reason is, if the user taps OK without changing the email, this method will still be called, but the identity won't be changed in `ZendeskHelper.refreshIdentity`.

Reply:
> I think it would be better if we had this in ZendeskHelper.setSupportEmail since it's where the identity change happens

I waffled on this actually and for me it came down to a coin toss.  I'm not opposed to moving it. 

> I'd also suggest checking if the email is actually changed

This was informed and deliberate, and reflected by naming the stat `_SET` vs `_CHANGED`. Maybe a bit too subtle of a distinction without an accompanying comment. The thought was this would better reflect usage of the dialog vs actually changing its value.  I could probably be convinced checking for a change is better.  Thoughts?

Reply:
> I waffled on this actually and for me it came down to a coin toss. I'm not opposed to moving it.

I personally prefer it if it was in the `ZendeskHelper` mostly because that's where the actual action is taken and also because we have other events for Zendesk there.

> This was informed and deliberate, and reflected by naming the stat _SET vs _CHANGED. Maybe a bit too subtle of a distinction without an accompanying comment. The thought was this would better reflect usage of the dialog vs actually changing its value. I could probably be convinced checking for a change is better. Thoughts?

As long it's a decision and not an oversight, I am good either way. You have a lot more experience looking at the analytics than I do, so I'll leave that decision to you :)